### PR TITLE
Ordeals of Black (Echoes of You)

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Adds in an alternate to white ordeals, black ordeals.
They are all copies of your aleph agent stats; just with more HP.
They do the exact same damage and move at the same speed, and have the exact same armor.

- [x] Paradise Lost, DF and Twilight Echo
- [ ] Lore writing

## Why It's Good For The Game
White ordeals are getting very stale. I am looking for a suitable replacement.

## Changelog
:cl:
add: Added Ordeals of Black
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/ordeal/black.dm [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+			melee_damage_lower = 184
+			melee_damage_upper = 184
+		if(WHITE_DAMAGE)
+			melee_damage_lower = 161
+			melee_damage_upper = 161
+		if(PALE_DAMAGE)
+			melee_damage_lower = 115
+			melee_damage_upper = 115
+
+	. = ..()
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/hostile/ordeal/black.dm:70]:
```suggestion
	return ..()
```

File: code/modules/mob/living/simple_animal/hostile/ordeal/black.dm [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+			attack_sound = 'sound/weapons/ego/justitia4.ogg'
+			melee_damage_lower = 86.25
+			melee_damage_upper =  86.25
+			combo = 0
+
+			var/turf/T = get_turf(target)
+			new /obj/effect/temp_visual/justitia_effect(T)
+			HurtInTurf(T, list(), 50, PALE_DAMAGE)
+
+	. = ..()
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/hostile/ordeal/black.dm:168]:
```suggestion
	return ..()
```

**Review by Gboster-0 — COMMENTED** [review]
le `return ..()` nitpicks

File: code/modules/mob/living/simple_animal/hostile/ordeal/black.dm [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+
+//Justitia
+/mob/living/simple_animal/hostile/ordeal/echo/justitia
+	icon_state = "justitia_echo"
+	icon_living = "justitia_echo"
+	rapid_melee = 4
+	melee_damage_type = PALE_DAMAGE
+	melee_damage_lower = 57.7
+	melee_damage_upper = 57.7
+	damage_coeff = list(RED_DAMAGE = 0.4, WHITE_DAMAGE = 0.0, BLACK_DAMAGE = 0.4, PALE_DAMAGE = 0.8)
```
[the remark this answers is not in the document]
Comment (Kitsunemitsu) [reply code/modules/mob/living/simple_animal/hostile/ordeal/black.dm:132]: Ah, Minor bug. Will fix.

File: code/modules/mob/living/simple_animal/hostile/ordeal/black/_black.dm [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+		return
+	var/mob/living/H = target
+	for(var/damage_type in list(WHITE_DAMAGE, BLACK_DAMAGE, PALE_DAMAGE))
+		H.apply_damage(melee_damage_lower, damage_type, null, H.run_armor_check(null, damage_type))
+
+
+/mob/living/simple_animal/hostile/ordeal/echo/distorted/attacked_by(obj/item/I, mob/living/user)
+	//the block
+	if(prob(30))
+		damage_coeff = list(RED_DAMAGE = 0.08, WHITE_DAMAGE = 0.12, BLACK_DAMAGE = 0.08, PALE_DAMAGE = 0.2)
```
Comment (tile4) [inline code/modules/mob/living/simple_animal/hostile/ordeal/black/_black.dm:279]:
```suggestion
		ChangeResistances(0.08, 0.12, 0.08, .2)
```

File: code/modules/mob/living/simple_animal/hostile/ordeal/black/_black.dm [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+		playsound(get_turf(src), 'sound/weapons/ego/shield1.ogg', 50, 0, 7)
+	else
+		damage_coeff = list(RED_DAMAGE = 0.2, WHITE_DAMAGE = 0.3, BLACK_DAMAGE = 0.2, PALE_DAMAGE = 0.5)
```
Comment (tile4) [inline code/modules/mob/living/simple_animal/hostile/ordeal/black/_black.dm:282]:
```suggestion
			ChangeResistances(0.2, 0.3, 0.2, .5)
```

**Review by tile4 — CHANGES_REQUESTED** [review]
This is kinda needed to prevent Distortion Echo from being invincible

File: code/modules/mob/living/simple_animal/hostile/ordeal/black/_black.dm (hunk 4 above)
Comment (Kitsunemitsu) [reply code/modules/mob/living/simple_animal/hostile/ordeal/black/_black.dm:279]: Thought I already did this, maybe I just didn't push it

**Comment by Kitsunemitsu** [comment]
Content is being moved to #2204

**Pull request closed without merging**